app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide(:title, t(".home")) %>
+    <h1><%= t(".home") %></h1>
+    <p>
+      <a href="https://www.railstutorial.org/"><em><%= t(".link_1") %></em></a> is a
```

Comment:
`is a` -> I18n

mà thay vì dùng `a` tag của html, thanh niên dùng `link_to` tag của rails hì